assets/pages/digital-subscription-landing/digitalSubscriptionLandingReducer.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { combineReducers } from 'redux';
+import type { CommonState } from 'helpers/page/commonReducer';
+import { type DigitalBillingPeriod } from 'helpers/billingPeriods';
+import {
+  ProductPagePlanFormReducerFor,
+  type State as FormState,
+} from 'components/productPage/productPagePlanForm/productPagePlanFormReducer';
+import promotionPopUpReducer, { type FindOutMoreState } from './components/promotionPopUpReducer';
+
+export type State = {
+  common: CommonState,
+  page: {
+    plan: FormState<DigitalBillingPeriod>,
+    promotion: FindOutMoreState
+  }
+};
+
+
+// ----- Export ----- //
+
+export default (promoInUrl: ?string) => {
+
+  const initialPeriod: ?DigitalBillingPeriod = promoInUrl === 'Monthly' || promoInUrl === 'Annual'
```

Comment:
I worry about the precedence of these operators! `||` vs `? :`

I'm sure you've tested it but some brackets around `(promoInUrl === 'Monthly' || promoInUrl === 'Annual')` would help me sleep at night

Reply:
`eslint` will remove them!